.travis.yml

```diff
@@ -2,7 +2,8 @@ language: go
 dist: trusty
 sudo: required
 go:
-  - 1.7.3
+  - 1.7.5
+  - 1.8.1
```

Comment:
FYI, you can now use `1.7.x` and `1.8.x` to target latest patches of Go in Travis. That way, there's no need to change `.travis.yml` when a new patch is released.

Reference: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use.